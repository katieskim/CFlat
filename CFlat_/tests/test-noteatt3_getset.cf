int main()
{
	note n;
	note v;
	tone t;
	octave o;
	rhythm r;

	n = ( /C-/ /4/ /s./ );

	t = n.gettone();
	o = n.getoctave();
	r = n.getrhythm();

	v.settone(t);
	printt(v.gettone());

	v.settone(/A/);
	printt(v.gettone());

	v.setoctave(o);
	printo(v.getoctave());

	v.setoctave(/5/);
	printo(v.getoctave());

	v.setrhythm(r);
	printr(v.getrhythm());

	v.setrhythm(/h/);
	printr(v.getrhythm());
	
	return 0;
}
