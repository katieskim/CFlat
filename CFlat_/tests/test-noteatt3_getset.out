C-
A
4
5
s.
h